components/dashboard/src/settings/Plans.tsx

```diff
@@ -711,6 +711,15 @@ export default function () {
                     </div>
                 )}
                 <div className="mt-4 flex justify-center space-x-3 2xl:space-x-7">{planCards}</div>
+                {assignedTs && (
+                    <InfoBox className="w-full mt-14 mx-auto">
```

Comment:
**thought**: I remember discussing issue back in https://github.com/gitpod-io/gitpod/issues/3948 and https://github.com/gitpod-io/gitpod/issues/5977. Given that a) team plans are fading away, b) team billing is the only option for new teams, and c) our future plans to switch to UBP, I don't think it's worth putting any effort in implementing the proposal in https://github.com/gitpod-io/gitpod/issues/5977#issuecomment-932128529.